.github/workflows/R-CMD-check.yaml

```
@@ -51,6 +40,28 @@ jobs:
 
       - uses: r-lib/actions/setup-pandoc@master
 
+      - name: Brew and macOS config
+        if: runner.os == 'macOS'
+        run: |
+          brew install pkg-config
+          brew install udunits
+          brew install gdal
+          cat <<EOT >> .Renviron
+          PKG_CONFIG_PATH=/usr/local/lib/pkgconfig/
+          PROJ_LIB=/usr/local/opt/proj/share/proj/
+          # for installing XML package from source
+          XML_CONFIG=/usr/local/opt/libxml2/bin/xml2-config
+          EOT
+          cat <<EOT >> .Rprofile
+          r <- getOption("repos")
+          r["CRAN"] <- "https://cran.rstudio.com"
+          r["rccp_drat"] <- "https://RcppCore.github.io/drat"
+          config_args <- c("sf" = "--with-proj-lib=/usr/local/lib/", "rgdal" = "--with-proj-lib=/usr/local/lib/ --with-proj-include=/usr/local/include/")
+          options(configure.args = config_args,
+                  install.packages.compile.from.source = "yes",
+                  repos = r)
+          EOT
```

Maybe this would be easier to read?
```suggestion
          (
          cat <<-EOT 
              PKG_CONFIG_PATH=/usr/local/lib/pkgconfig/
              PROJ_LIB=/usr/local/opt/proj/share/proj/
              # for installing XML package from source
              XML_CONFIG=/usr/local/opt/libxml2/bin/xml2-config
          EOT
          ) >> .Renviron
          
          (
          cat <<-EOT 
              r <- getOption("repos")
              r["CRAN"] <- "https://cran.rstudio.com"
              r["rccp_drat"] <- "https://RcppCore.github.io/drat"
              config_args <- c("sf" = "--with-proj-lib=/usr/local/lib/", "rgdal" = "--with-proj-lib=/usr/local/lib/ --with-proj-include=/usr/local/include/")
              options(configure.args = config_args,
                      install.packages.compile.from.source = "yes",
                      repos = r)
          EOT
          ) >> .Rprofile
```

---

I've never seen heredocs written that way before, just tested on my computer and it didn't seem to work...?

---

Works on my side (bash ubuntu), but let's leave it like that. We can improve readability later :)

---

Sounds good to me, I have zsh as my shell, that may be it